src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -156,21 +152,23 @@ private static void cloneAndAnalyzeRepos(List<RepoConfiguration> configs, String
             if (clonedRepoLocation == null) {
                 handleCloningFailed(configs, currRepoLocation);
             } else {
-                analyzeRepos(outputPath, configs, repoLocationMap.get(clonedRepoLocation),
-                        repoCloner.getCurrentRepoDefaultBranch());
+                generatedFiles.addAll(analyzeRepos(outputPath, configs, repoLocationMap.get(clonedRepoLocation),
+                    repoCloner.getCurrentRepoDefaultBranch()));
```

Comment:
We are following the OSS coding standards. So indentation for wrapped lines should be 8 spaces

Reply:
Done.